docs/user/04-metrics.md

```diff
@@ -670,7 +670,38 @@ If you want to use them for debugging and diagnostic purposes, you can activate
 > [!NOTE]
 > Diagnostic metrics are only available for inputs `prometheus` and `istio`. Learn more about the available [parameters and attributes](resources/05-metricpipeline.md).
 
-### 11. Deploy the Pipeline
+### 11. Activate Envoy Metrics
```

Comment:
Why not describing it in the section where the istio input is explained? That is very specific for Istio and decoupling the two topics sounds wrong to me..

Reply:
moved into istio input section